src/components/main/adaptive/feature/CBD/ClubDetail.jsx

```
@@ -28,53 +36,279 @@ const linkifyText = (text) => {
+              <polyline points="9 18 15 12 9 6" />
+            </svg>
+          </button>
+        )}
+      </div>
+    </div>
+  );
+};
+
+// ─── 모바일 레이아웃 ──────────────────────────────────────────────────────────
+
+const MobileLayout = ({ title, status, vendors, categories, content, linkUrl, attachments }) => {
```

ESLint

[error] 'linkUrl' is defined but never used.

(no-unused-vars)